packages/mask/src/plugins/Tips/SNSAdaptor/components/WalletItem.tsx

```diff
@@ -0,0 +1,185 @@
+import { useReverseAddress, useWeb3State } from '@masknet/plugin-infra/web3'
+import { FormattedAddress, useSnackbarCallback } from '@masknet/shared'
+import { makeStyles } from '@masknet/theme'
+import { ChainId, isSameAddress, useWallets } from '@masknet/web3-shared-evm'
+import { Link, Typography } from '@mui/material'
+import { useMemo } from 'react'
+import { useCopyToClipboard } from 'react-use'
+import { useI18N } from '../../../../utils'
+
+const useStyles = makeStyles()((theme) => ({
+    currentAccount: {
+        padding: theme.spacing(1.5),
+        display: 'flex',
+        border: `1px solid ${theme.palette.background.default}`,
+        borderRadius: 8,
+        alignItems: 'center',
+        justifyContent: 'space-between',
+    },
+    accountInfo: {
+        fontSize: 16,
+        flexGrow: 1,
+        marginLeft: theme.spacing(1),
+    },
+    infoRow: {
+        display: 'flex',
+        alignItems: 'center',
+    },
+    accountName: {
+        fontWeight: 700,
+        fontSize: 14,
+        marginRight: 6,
+    },
+    address: {
+        fontSize: 10,
+        marginRight: theme.spacing(1),
+        color: theme.palette.text.secondary,
+        display: 'inline-block',
+    },
+    link: {
+        color: theme.palette.text.primary,
+        fontSize: 14,
+        display: 'flex',
+        alignItems: 'center',
+    },
+    linkIcon: {
+        width: 16,
+        height: 16,
+        marginRight: theme.spacing(1),
+    },
+    defaultBtn: {
+        fontSize: 14,
+        fontWeight: 'bold',
+        cursor: 'pointer',
+    },
+    delIcon: {
+        width: 20,
+        cursor: 'pointer',
+    },
+    defaultBadge: {
+        padding: '2px 4px',
+        borderRadius: 4,
+        fontSize: 12,
+        backgroundColor: 'rgba(28, 104, 243, 0.1)',
+        color: '#1C68F3',
+        fontWeight: 700,
+        marginLeft: 4,
+    },
+    domain: {
+        fontSize: 16,
+        lineHeight: '18px',
+        marginLeft: 6,
+        padding: 4,
+        borderRadius: 8,
+        backgroundColor: '#ffffff',
+        color: theme.palette.common.black,
+    },
+}))
+
+interface WalletItemProps {
+    address: string
+    isDefault?: boolean
+    canDelete?: boolean
+    onDelete?: any
+    fallbackName?: string
+    nowIdx: number
+    setAsDefault?: (idx: number) => void
+}
+
+export function WalletItem({
+    address,
+    isDefault,
+    canDelete,
+    fallbackName,
+    setAsDefault,
+    onDelete,
+    nowIdx,
+}: WalletItemProps) {
+    const { classes } = useStyles()
+    const { t } = useI18N()
+    const [, copyToClipboard] = useCopyToClipboard()
+    const { value: domain } = useReverseAddress(address)
+    const { Utils } = useWeb3State() ?? {}
+    const onCopy = useSnackbarCallback(
+        async (ev: React.MouseEvent<HTMLAnchorElement>) => {
+            ev.stopPropagation()
+            copyToClipboard(address)
```

Comment:
```suggestion
            copyToClipboard(formatEthereumAddress(address))
```

Reply:
[done in ](https://github.com/DimensionDev/Maskbook/pull/6121)